test/jenkins/criu-fault.sh

```diff
@@ -9,7 +9,7 @@ prep
 ./test/zdtm.py run -t zdtm/static/maps00 --fault 3 --report report -f h || fail
 
 # FIXME: fhandles looks broken on btrfs
-cat /proc/self/mountinfo | grep -P "/.* / " | grep -q btrfs || NOBTRFS=$?
+< /proc/self/mountinfo grep -P "/.* / " | grep -q btrfs || NOBTRFS=$?
```

Comment:
Things like this can be changed to `grep -P "/.* / " /proc/self/mountinfo`. The extra `<` seems unnecessary in this case. There are couple of cases here in this PR like this. Not all of them.

Reply:
It seems that most of the cases in this PR have the extra redirection operator. I have incorporated them in the latest commit.